src/v3/src/components/ActionPending/ActionPending.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box } from '@mui/material';
+import { Typography } from '@okta/odyssey-react-mui';
+import { h } from 'preact';
+import { useEffect, useRef } from 'preact/hooks';
+
+import { useWidgetContext } from '../../contexts';
+import { ActionPendingElement, UISchemaElementComponent } from '../../types';
+
+/**
+ * Render component used to indicate to the user that they should wait for an action to complete.
+ * The component shows text describing the action that is pending.
+ */
+const ActionPending: UISchemaElementComponent<{
+  uischema: ActionPendingElement
+}> = (
+  { uischema: { options } },
+) => {
+  const titleRef = useRef<HTMLTitleElement>(null);
+  const { widgetProps } = useWidgetContext();
+  const { features: { autoFocus = false } = {} } = widgetProps;
+
+  useEffect(() => {
+    if (!autoFocus) {
```

Comment:
Is this condition backwards?

Reply:
My guess is, if they configure autoFocus to be disabled, we want this to always be focused? If so, how will that play with [this](https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/components/Title/Title.tsx#L30-L35)? Maybe this gets loaded onto the page after the title so this takes precedence?

Reply:
`Title` and this component don't appear at the same time in any of the scenarios, and this text was previously treated the same way as a title heading in the Gen2 widget, based on the testcafe test objects, so I am using the same autofocus behavior given that it will face the same type of a11y issue as `Title` as the first text element we would want a screenreader to focus on in the view.

Reply:
And yes, to @glenfannin-okta 's point, this is the intended logic (I copied it from `Title.tsx`) -- so if no element is autofocused with the widget config, we want to move focus to the view title so the screenreader will jump to it when a new view loads.

Reply:
Out of scope for this bug fix, but we want to consider moving all of this logic into the `setFocusOnFirstElement` transformer.